docs/Halogen-Events.md

```diff
@@ -64,6 +64,14 @@ This function provides an alternative to making two nested calls to `pure`:
 onclick (input \_ -> Input)
 ```
 
+#### `withEventHandlerT`
+
+``` purescript
+withEventHandlerT :: forall i m e. (e -> EventHandlerT m i) -> e -> EventHandler (m i)
```

Comment:
That seems like it would be the cleaner approach, right? Then users would never have to wrap like this, whether they want the `T` version or not.